src/apps/Console/ConsoleFeedback.js

```diff
@@ -21,325 +28,269 @@ export default function ConsoleFeedback({
           display: flex;
           align-items: center;
           justify-content: center;
-          padding: ${GU * 6}px 0 ${GU * 6}px 0;
+          padding: ${6 * GU}px 0 ${6 * GU}px 0;
         `}
       >
         <LoadingRing />
         <span
           css={`
             margin-left: ${1 * GU}px;
-            ${textStyle('body2')}
+            ${textStyle('body2')};
           `}
         >
-          Executing Command...
+          Executing command...
         </span>
       </div>
     )
   }
+
   if (currentParsedCommand.length < 2) {
     return (
       <>
-        <p
-          css={`
-            ${textStyle('body2')}
-          `}
-        >
-          You can interact with this organization by using the following CLI
+        <Paragraph>
+          You can interact with this organization by using the following console
           commands:
-        </p>
+        </Paragraph>
         <div
           css={`
-            width: 100%;
-            margin-top: ${GU}px;
+            overflow: auto;
           `}
         >
           {['Exec', 'Act'].map(command => (
```

Comment:
What do you think of lifting this to an array of `AVALABLE_COMMANDS` so it's easier to change later and it's not embedded in the feedback code itself?

Reply:
Done in 75ac34a.